src/App.jsx

```diff
@@ -0,0 +1,203 @@
+//  TODO APP : Designed to cheer for each little accomplishment we make as we progress toward our goals - like building an app. So this app cheers for you when you complete each task and celebrates with you when you complete all of your tasks.
+
+import { BrowserRouter, Routes, Route, Link } from 'react-router-dom';
+import { MdOutlinePlaylistAddCheck, MdArrowBackIosNew } from 'react-icons/md';
+import React, { useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+
+import AddTodoForm from './components/AddTodoForm';
+import TodoList from './components/TodoList';
+import CompletedTodos from './components/CompletedTodos';
+import { ConfiguredToast } from './components/ConfiguredToast';
+
+import 'react-toastify/dist/ReactToastify.css';
+
+const airtableName = 'Todos';
+const airtableView = '?view=Grid%20view';
+const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${airtableName}/`;
+
+function App() {
+	const [todoList, setTodoList] = useState([]);
+	const [isLoading, setIsLoading] = useState(false);
+	const [isMuted, setIsMuted] = useState(false);
+
+	// GET TODOS FROM (AIRTABLE) DB
+	useEffect(() => {
+		const timeoutId = setTimeout(() => {
+			loadTodos();
+		}, 3000);
```

Comment:
This works as it is taking advantage of the cleanup function but can you think of some other way to prevent the first firing? There are other constructs that are more appropriate as setTimeout is used to delay an action, where in this case, your goal is to prevent an action.